e2e/donation-modal.spec.ts

```diff
@@ -313,6 +313,76 @@ test.describe('Donation modal appearance logic - Certified user claiming a new b
     await completeFrontEndCert(page, 1);
     await expect(donationModal).toBeHidden();
   });
+
+  test("should not appear if the user has completed a new FSD block, but the block's module is not completed", async ({
+    page
+  }) => {
+    await page.goto(
+      '/learn/full-stack-developer/review-basic-html/basic-html-review'
+    );
+
+    await page.getByRole('checkbox', { name: /Review/ }).click();
+    await page.getByRole('button', { name: 'Submit', exact: true }).click();
+    await page.getByRole('button', { name: /Submit and go/ }).click();
+
+    const donationModal = page
+      .getByRole('dialog')
+      .filter({ hasText: 'Become a Supporter' });
+    await expect(donationModal).toBeHidden();
+  });
+});
+
+test.describe('Donation modal appearance logic - Certified user claiming a new module', () => {
+  test.use({ storageState: 'playwright/.auth/certified-user.json' });
+  execSync('node ./tools/scripts/seed/seed-demo-user --almost-certified-user');
+
+  test('should appear if the user has just completed a new module', async ({
+    page
+  }) => {
+    test.setTimeout(40000);
```

Comment:
@ojeytonwilliams, we discussed this the other day.
Instead of waiting for the modal, should I just add different test ids for different test ids for different donation modals and look for those?

Reply:
If necessary, we can wait for the animation (it's not ideal, though). If you can test what you're interested in without waiting, that would be great.